Resources/views/frontend/index/header.tpl

```diff
@@ -3,7 +3,7 @@
 {block name='frontend_index_header_javascript_modernizr_lib'}
     {block name='frontend_index_header_javascript_modernizr_lib_paypal_unified_checkout_button'}
         {if $paypalUnifiedEcCartActive || $paypalUnifiedEcDetailActive || $paypalUnifiedEcLoginActive || $paypalUnifiedUseInContext}
-            <script src="https://www.paypalobjects.com/api/checkout.js"></script>
+            <script src="https://www.paypalobjects.com/api/checkout.min.js"></script>
```

Comment:
the unminified version could still be used if `paypalUnifiedModeSandbox` is set.

Reply:
sounds legit. thanks :-)